code/game/objects/items/clown_items.dm

```diff
@@ -102,8 +102,13 @@
 
 	else if(ishuman(target) && user.zone_selected == BODY_ZONE_PRECISE_MOUTH)
 		var/mob/living/carbon/human/H = user
-		user.visible_message("<span class='warning'>\the [user] washes \the [target]'s mouth out with [src.name]!</span>", "<span class='notice'>You wash \the [target]'s mouth out with [src.name]!</span>") //washes mouth out with soap sounds better than 'the soap' here
-		H.lip_style = null //removes lipstick
+		if(user.zone_selected == "mouth")
+			user.visible_message("<span class='warning'>\the [user] washes \the [target]'s mouth out with [src.name]!</span>", "<span class='notice'>You wash \the [target]'s mouth out with [src.name]!</span>") //washes mouth out with soap sounds better than 'the soap' here
+			H.lip_style = null //removes lipstick
+			H.adjust_hygiene(5) //it kinda works i guess

```

Comment:
So all I need to do to stay clean is just wash my mouth out with soap, 20 times?

Reply:
if you are that autistic, sure.

Reply:
***E̡A͚͖̦̲͔͚͉͝T ̡̱͍̜̦͙ͅT͕͟H͍E̻̲̝̹͚ ͠S̗͝ͅO̹̦̳̝͉ͅĄ̫P̮̱̝̪͓̠̟͟ ~~CON̶̥̗͕S̩UM̹͉̖E͡~~*** *S͏̬͙̟͙̝̘̜O̞A̬͈P* `i̥̝̙͖̭̤̤͞n̟̣s̙͖̲̪̕ͅe̻r̬͉t̘̬̦̺͈̜` ~~***S͎̠̳̱͜O̝̯̺͉̰̗̹͝A̬̱̞P̟͕̝ͅ***~~ C̨̤̜L̝̣̻̱E͍̜̤̠̰A͚̰̮̤̝̤ͅN̲̥͖S̳͖̟͜I͏N̻̲G̜̻̺ **PR͎͞O̟̺̪͎D͎͞U͠C̴͍̳̘T͇̯͕̹͙̝̺S͓̕**